### PR TITLE
don't override falsy values in process.env

### PR DESCRIPTION
**Pull request by legendar**

### CODE REVIEW

**Comment by ericrallen** [comment]
Based on some quick testing with another project this seems to resolve #18

**Comment by ericrallen** [comment]
@motdotla Any chance of this getting merged in?

Is there anything else you'd want to see in a PR like this to get it across the finish line or is there a different way you'd see the fix being implemented?

File: lib/main.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var dotenvExpand = function (config) {
       var key = match.replace(/\$|{|}/g, '')
 
       // process.env value 'wins' over .env file's value
-      var variable = process.env[key] || config.parsed[key] || ''
+      var variable = process.env.hasOwnProperty(key) ? process.env[key] : (config.parsed[key] || '')
```
Comment (ericrallen) [inline lib/main.js]: Should the two lines you updated be abstracted into a method given the similarities between the two?

Comment (legendar) [reply lib/main.js]: sorry, this is not code refactoring PR. it just fixing an issue. 
you can make you own fork and make some additional changes, but it looks like there no chance to merge it.

**Comment by motdotla** [comment]
This looks good. Thank you guys, good catch. Merging.

**Comment by motdotla** [comment]
This is available in 5.0.0 now https://www.npmjs.com/package/dotenv-expand

**Pull request merged**